frontend/web/components/pages/AnimeDetails.js

```diff
@@ -28,25 +29,56 @@ class AnimeDetails extends React.Component {
   render () {
     const { animeCache } = this.props;
     let content = null;
-    let details = animeCache[this.props.params.id];
+    let details = animeCache[this.props.params.id]
     if (details) {
-      console.log('got details');
-      let averageScore = (Math.round(parseInt(details.average_score, 10)))/10;
-      let description = details.description.replace(/<[^>]*>/ig, "");
+      console.log('got details')
+      console.log(details);
+      let averageScore = (Math.round(parseInt(details.average_score, 10)))/10
+      let description = details.description.replace(/<[^>]*>/ig, "")
+      let characters = []
+      if (details.characters.length > 0) {
+        characters = details.characters.slice(0, 7)
+          .map((charDetail, i) => {
+            return <li className="character-card" key={i}>
```

Comment:
Since you're just returning here, you can remove the `{}` and do an implicit return so:

```js
.map((charDetail, i) => ( // still using parens to wrap JSX
  <li>
    //...
  </li>
)
```